README.md

```diff
@@ -9,112 +9,46 @@
 [![Renovate enabled](https://img.shields.io/badge/renovate-enabled-brightgreen.svg)](https://renovatebot.com/)
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 
```

Comment:
`latest release` badge above needs to be updated (in two place) - it currently points to module template repo

I also suggest updating the `Incubating` badge to `Stable` since after we merge we will have a stable release (see all available source code for badges at https://terraform-ibm-modules.github.io/documentation/#/badge-status)